js_modules/dagit/packages/core/src/workspace/RepositoryLocationsList.tsx

```diff
@@ -17,12 +17,14 @@ import {Tooltip} from '../ui/Tooltip';
 
 import {RepositoryLocationNonBlockingErrorDialog} from './RepositoryLocationErrorDialog';
 import {RepositoryRemoteLocationLink} from './RepositoryRemoteLocationLink';
-import {WorkspaceContext} from './WorkspaceContext';
-import {RootRepositoriesQuery_workspaceOrError_Workspace_locationEntries as LocationOrError} from './types/RootRepositoriesQuery';
+import {WorkspaceContext, WorkspaceState} from './WorkspaceContext';
 
 const TIME_FORMAT = {showSeconds: true, showTimezone: true};
 
-const LocationStatus: React.FC<{location: string; locationOrError: LocationOrError}> = (props) => {
+const LocationStatus: React.FC<{
+  location: string;
+  locationOrError: WorkspaceState['locationEntries'][0];
```

Comment:
Use the same type as the one referenced by `WorkspaceContext`? Using the `[0]` makes me feel a little itchy.

Reply:
Yeah good call - I'll export a few more things from WorkspaceContext so it's easier to consume this!